static/bidder-info/kiviads.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "http://endpoint1.kiviads.com/pserver"
+maintainer:
+  email: "prebid@kiviads.com"
+capabilities:
+  site:
+    mediaTypes:
+    - banner
+    - video
+    - native
+  app:
+    mediaTypes:
+    - banner
+    - video
+    - native
+userSync:
+  redirect:
+    url: "https://sync.kiviads.com/pserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
This always returns "No GDPR/CCPA consent" and http 400. 
This is test url I use: https://sync.kiviads.com/pserver?gdpr=&gdpr_consent=&us_privacy=&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dkivi%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D

Reply:
Thx, we will fix it soon. The url https://sync.kiviads.com/pserver remains valid

Reply:
Hello, this still doesn't work. Do I miss something?

Reply:
Here is updated URL I tried, same result: https://sync.kiviads.com/pserver?gdpr=&gdpr_consent=CPfCRQAPfCRQAAAAAAENCgCAAOAAAAAAAAAAAQAAAAAEAIAAAAAAAAAA&us_privacy=1NYN&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dkivi%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D

Reply:
@VeronikaSolovei9 We do not sync if the request is from the European region and gdpr consent is not valid

Reply:
@KiviAds would you mind providing a working example just so we can see it working before approving? We're not in Europe.

Reply:
@VeronikaSolovei9 @bsardo thx, we fixed bug, working link - https://sync.kiviads.com/pserver?gdpr=&gdpr_consent=CPfCRQAPfCRQAAAAAAENCgCAAOAAAAAAAAAAAQAAAAAEAIAAAAAAAAAA&ccpa=1NYN&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dkivi%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D

Reply:
It works, thank you!